### DISCUSSION TITLE
Ask Your Questions

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Gordian#84] [Questions & Answers]
How do you ask questions about Gordian apps?

What sort of questions should you include?

**Comment by shannona** [comment]
Just write any questions that you might have here in the Q&A category. They can include things that confuse you, things that you're not sure if they're bugs, or just questions about philosophy and design choices.

Then, someone can answer them (like this answer does).

Feel free to vote on answers too: the best ones will rise to the top.

**Reply by yfrbl** [reply]
Hi @ChristopherA, 
I installed Gordianserver for macOS and fully synced testnet. Now, when I would like to access bitcoin-cli (like it's mentioned in the blockchaincommons course), it doesn't work. How do I fix it?

Really appreciate your help!

**Comment by aido** [comment]
Hi,

Is BlockchainCommons' proposed Collaborative Seed Recovery (CSR) a similar methodology to [Ledger Recover](https://www.ledger.com/recover)?

[Ledger Recover FAQ](https://support.ledger.com/hc/en-us/articles/9579368109597-Ledger-Recover-FAQs)

**Reply by aido** [reply]
Ledger have released the white paper for their Recover service:

https://github.com/LedgerHQ/recover-whitepaper

Looks like they are using Pedersen Verifiable Secret Sharing.

**Reply by aido** [reply]
Ledger have released more details of their Recover service:

[Part 1: Genesis of Ledger Recover – Self custody without compromise](https://www.ledger.com/blog/part-1-genesis-of-ledger-recover-self-custody-without-compromise)

[Part 2: Genesis of Ledger Recover – Securely distributing the shares](https://www.ledger.com/blog/part-2-genesis-of-ledger-recover-securely-distributing-the-shares)

[Part 3: Genesis of Ledger Recover – Avoiding collusion and leaks](https://www.ledger.com/blog/part-3-genesis-of-ledger-recover-avoiding-collusion-and-leaks)

[Part 4: Genesis of Ledger Recover – Controlling access to the backup: Identity verification](https://www.ledger.com/blog/part-4-genesis-of-ledger-recover-controlling-access-to-the-backup-identity-verification)

[Part 5: Genesis of Ledger Recover – Operational Security](https://www.ledger.com/blog/part-5-genesis-of-ledger-recover-operational-security)

[Part 6: Genesis of Ledger Recover – Threat analysis/evaluation](https://www.ledger.com/blog/part-6-genesis-of-ledger-recover-security-analysis)

**Reply by aido** [reply]
Ledger have released information on the tools that allow an individual implement their own Ledger Recover shard backup provider:

[How to Recover](https://github.com/LedgerHQ/blue-loader-python/blob/master/ledgerblue/HowToRecover.md)


I had been writing an application that implements SSKR on Ledger devices and was concerned that the Ledger Recover tools would  overlap with the application or even make it obsolete. I had put development on hold but now I see that there is no overlap so I plan on  continuing with app development.

https://github.com/aido/app-sskr-check/discussions/11
